src/renderer/components/welcome.tsx

```diff
@@ -137,95 +133,124 @@ export class Welcome extends React.Component<
   private platformIcon = (platform: string) => {
     switch (platform) {
       case 'win32':
-        return <WindowsOutlined style={iconStyle} />;
+        return <WindowsFilled style={iconStyle} />;
       case 'darwin':
-        return <AppleOutlined style={iconStyle} />;
+        return <AppleFilled style={iconStyle} />;
       case 'linux':
-        return <QqOutlined style={iconStyle} />;
+        return <LinuxOutlined style={iconStyle} />;
       case 'ios':
-        return <MobileOutlined style={iconStyle} />;
+        return <MobileFilled style={iconStyle} />;
       case 'android':
-        return <AndroidOutlined style={iconStyle} />;
+        return <AndroidFilled style={iconStyle} />;
       default:
-        return <SlackOutlined style={iconStyle} />;
+        return <SlackCircleFilled style={iconStyle} />;
     }
   };
 
   public renderSuggestions(): JSX.Element | null {
     const { openFile } = this.props.state;
 
-    if (this.props.state.suggestions?.length) {
+    const today: Suggestion[] = [];
+    const thisWeek: Suggestion[] = [];
+    const stale: Suggestion[] = [];
+
+    const { suggestions } = this.props.state;
+
+    for (const item of suggestions) {
+      if (isToday(item.mtimeMs)) {
+        today.push(item);
+      } else if (isThisWeek(item.mtimeMs)) {
+        thisWeek.push(item);
+      } else {
+        stale.push(item);
+      }
+    }
+
+    const list = (data: Suggestion[]) => {
+      return (
+        <List
+          itemLayout="horizontal"
+          dataSource={data}
+          renderItem={(item) => {
+            const openItem = (e: React.MouseEvent) => {
+              e.preventDefault();
+              openFile(item.filePath);
+            };
+            const deleteItem = (e: React.MouseEvent) => {
+              e.stopPropagation();
+              this.deleteSuggestion(item.filePath);
+            };
+
+            return (
+              <List.Item
+                className="welcome__suggestion-list-item"
+                actions={[
+                  <Button
+                    danger={true}
+                    key="list-delete"
+                    onClick={deleteItem}
+                    icon={<DeleteOutlined />}
+                  >
+                    Delete
+                  </Button>,
```

Comment:
Could we make this button only visible if you hover over the list item?

Reply:
[b3b3258](https://github.com/tinyspeck/sleuth/pull/170/commits/b3b325813d3aca5a5533763cc5b8f2e3a72076d2)